{"name": "Test login with incorrect login", "status": "passed", "start": 1721592696386, "stop": 1721592696529, "uuid": "48d93d3f-555a-4fd3-ba21-09a27f484118", "historyId": "b427a8a34d9d3352ea1bfa90a6515a8d", "testCaseId": "b427a8a34d9d3352ea1bfa90a6515a8d", "fullName": "tests.test_login.TestLoginCourier#test_login_with_incorrect_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "LAPTOP-B1V9C35F"}, {"name": "thread", "value": "8296-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}